exercises/allergies/allergies_test.rb

```diff
@@ -47,24 +55,49 @@ def test_allergic_to_eggs_and_peanuts
     assert_equal %w(eggs peanuts), allergies.list
   end
 
+  def test_allergic_to_more_than_eggs_but_not_peanuts
+    skip
+    allergies = Allergies.new(5)
+    assert_equal %w(eggs shellfish), allergies.list
+  end
+
   def test_allergic_to_lots_of_stuff
     skip
     allergies = Allergies.new(248)
-    expected = %w(strawberries tomatoes chocolate pollen cats)
-    assert_equal expected, allergies.list
+    assert_equal %w(strawberries tomatoes chocolate pollen cats), allergies.list
```

Comment:
I wonder about keeping the expectation and assertion on separate lines, since the lines get quite long when the allergy list is long.
What do you think?

Reply:
I didn't even notice it when I generated the test, but I definitely like it on separate lines.